vars/gangplank.groovy

```diff
@@ -133,8 +133,12 @@ def runGangplank(params = [:]) {
     }
 }
 
-def _finalize() {
-    _runGangplank("gangplank pod -A finalize")
+def _finalize(params = [:]) {
+    gangplankCmd = "gangplank pod "
+    gangplankCmd = _getImage(params, gangplankCmd)
+    gangplankCmd = _getBucket(params, gangplankCmd)
```

Comment:
This stood out to me, should this be  `+=` instead of `=`. I'm not too familiar with groovy so correct me if I'm wrong.

Reply:
Oh nvm I see how it works now.